sirepo/sim_data/__init__.py

```diff
@@ -357,8 +357,7 @@ def lib_file_resource_path(cls, path):
         return sirepo.resource.file_path(
             _TEMPLATE_RESOURCE_DIR,
             cls.sim_type(),
-            cls._LIB_RESOURCE_DIR,
-        ).join(path)
+        ).join(cls._LIB_RESOURCE_DIR, path)
```

Comment:
Concerned about this solution. I'm not sure it fixes the problem. If someone creates, say, `template/<some-type>` and only wants to have the examples augmented, then it will be found, and there won't be any lib files in it (because the lib dir doesn't exist).

Reply:
Shoot, that's true. Need to think some more...

Reply:
91e5a986f thoughts?